### DISCUSSION TITLE
HTTP Listener: is there any limitation on the requests received by a channel?

### DISCUSSION THREAD

**Discussion by IbrahimAlhaj9** [nextgenhealthcare/connect#5618] [General]
Hi Everyone, 
I need to Know how many http requests can a channel or mirth as a whole received by a second? Is there any limitations?

Thank you

**Comment by pacmano1** [comment]
Sure there are limits, but you are going to need to sort that out with testing.    Factors include your thread settings, your backend hardware, what you are actually doing with the listener, your acceptable response time to a call, etc.

**Reply by IbrahimAlhaj9** [reply]
thank you, I need to setup many microservices instances(about 100) which will send each one more than 50 requests to mirth per seconds,  
So before starting, I want to know the reception limits of mirth.

**Comment by IbrahimAlhaj9** [comment]
Any idea about this? Thank you

**Comment by kirbykn2** [comment]
Ibrahim,

The limits are dependent on your specific configuration.  System resources,
Mirth channel configuration, code complexity, etc.

On Sat, Jan 21, 2023 at 1:29 AM Ibrahim Al Haj ***@***.***>
wrote:

> Any idea about this? Thank you
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/nextgenhealthcare/connect/discussions/5618#discussioncomment-4743545>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APRXWDYEFYGEWYALACQRJGLWTN625ANCNFSM6AAAAAAT655ZRY>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: <nextgenhealthcare/connect/repo-discussions/5618/comments/4743545@
> github.com>
>


-- 
Best,

Kirby Knight
| 231.735.4650 | ***@***.***

**Reply by IbrahimAlhaj9** [reply]
okk Thank you
